### PR TITLE
Export to iCal and GCal

### LINKED ISSUE #23: Export to iCal with proper weekly repetition and semester duration

### PR DESCRIPTION
**Pull request by G-ram**
Should close #23. Revised based on Nate's suggestions.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -85,9 +85,11 @@
   <script src="//code.angularjs.org/1.0.4/angular-cookies.min.js"></script>
   <script src="lib/lib.min.js"></script>
   <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
+  <script src="http://rawgithub.com/eligrey/FileSaver.js/master/FileSaver.js"></script>
```
Comment (brishin) [inline public/index.html]: Could you include these in bower.json and then concatenate them into lib.min.js (you'll have to edit `bower.json` and `Gruntfile.coffee`?

**Comment by brishin** [comment]
Also woah if possible please [squash commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) and then force push.

Also please [merge in upstream](https://help.github.com/articles/syncing-a-fork) so that the PR can merged automatically. I think some changes have been made to master recently.

**Comment by brishin** [comment]
bumping

**Comment by natebrennand** [comment]
@G-ram, we've updated our data so we're going to start promoting Courses for the next semester. If you could squash your commits together and do Brian's one edit we'd love to launch this feature

**Pull request closed without merging**